static/templates/index.html

```diff
@@ -41,6 +41,7 @@
     </style>
     <script src="{{root}}/skin/isotope.pkgd.min.js?KIWIXCACHEID" defer></script>
     <script src="{{root}}/skin/iso6391To3.js?KIWIXCACHEID"></script>
+    <script>const enableViewer = {{enable_viewer}};</script>
```

Comment:
Why not using the `viewer_setting.js` here ? (which could be renamed to `setting.js`)
It would be a central place indicating the configuration of the server.

Reply:
This is safer because it doesn't depend on the order of loading of `viewer_settings.js` and `index.js` (in which `enableViewer` is used).

Reply:
Ok